commands/intent.go

```diff
@@ -0,0 +1,29 @@
+package commands
+
+const (
+	// DefaultIntent Default Intent Type
+	DefaultIntent = iota
```

Comment:
I realize I'm shamelessly stealing Alexa terminology here, but 🤷‍♂️.

Reply:
I don't think Alexa coined it